charts/rstudio-workbench/values.yaml

```diff
@@ -463,14 +463,28 @@ config:
       default-session-cluster: Kubernetes
     vscode.conf:
       enabled: 1
-      exe: /opt/code-server/bin/code-server
-      args: --host=0.0.0.0
+      session-timeout-kill-hours: 12
```

Comment:
I probably missed some comms on this, but I thought we weren't going to enable this setting by default. I guess it's probably fine to make the timeout period this long by default though.

Reply:
I thought we agreed with setting it to a large value. @cm421 can you confirm?

Reply:
By default it is disabled in Workbench's settings, but for our helm charts we decided to set it to 12. It can always be changed if someone doesn't like it

Reply:
That makes sense!